reason-react-native/src/apis/Event.md

```diff
@@ -112,10 +112,10 @@ type pressEventPayload = {
 
 type pressEvent = responderSyntheticEvent(pressEventPayload);
 
-type position = {
+type contentOffset = {
   .
-  "y": float,
   "x": float,
+  "y": float,
```

Comment:
Don't know why y came before x?

Reply:
I followed the order in RN codebase, so there's no good reason for it.